src/query/service/src/spillers/spiller.rs

```diff
@@ -157,8 +157,10 @@ impl Spiller {
         let columns = data.columns().to_vec();
         let mut columns_data = Vec::with_capacity(columns.len());
         for column in columns.into_iter() {
-            let column = column.value.as_column().unwrap();
```

Comment:
There are a few other places that are also written this way, and are also likely to panic, Better change them together.

Reply:
agree with you.